### PR TITLE
Merge Golem-Verificator back to the core

### PR DESCRIPTION
**Pull request by Krigpl**
It doesn't make sense as a stand-alone repo.
Simply moved all the state from 1.7.0
No extra clean up at this stage.

### CODE REVIEW

**Comment by Krigpl** [comment]
catping

File: golem/verificator/docker/blender/images/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Use an official Python runtime as a parent image
```
Comment (nieznanysprawiciel) [inline golem/verificator/docker/blender/images/Dockerfile:1]: This is dockerfile that produces image_metrics image (this is the newest version). It is duplicated in golem/apps/rendering/resources/images/ (here resides old version, that isn't used anymore).
Remove old version.

**Review by mfranciszkiewicz — APPROVED** [review]

**Review by nieznanysprawiciel — CHANGES_REQUESTED** [review]

File: golem/verificator/docker/blender/images/Dockerfile (hunk 1 above)
Comment (mfranciszkiewicz) [reply golem/verificator/docker/blender/images/Dockerfile:1]: I'd suggest moving the latest to `golem/apps/rendering/resources/images`, so all images can be built by Golem automatically

Comment (Krigpl) [reply golem/verificator/docker/blender/images/Dockerfile:1]: I stated in the summary that this is supposed to be a simple code move, not a cleanup. There's much more to be cleaned, I'd rather not do it in the same PR.

Comment (nieznanysprawiciel) [reply golem/verificator/docker/blender/images/Dockerfile:1]: Moving these files should be simple move too ;)
But ok....
I approve this PR.

**Review by nieznanysprawiciel — APPROVED** [review]

**Pull request merged**